addons/mail/static/src/new/composer/emoji_picker.js

```diff
@@ -55,7 +55,17 @@ export class EmojiPicker extends Component {
             this.emojis = emojisData;
             this.state.categoryId = this.categories[0].sortId;
         });
-        onMounted(() => this.inputRef.el.focus());
+        onMounted(() => {
+            this.gridRef.el.onscroll = () => {
+                let coords = this.gridRef.el.getBoundingClientRect(); //Get coords of the emoji-picker-content element
+                let res = document.elementFromPoint(coords.x + 15, coords.y + 15); //Gets the top-most element, emojis are always the top-most elements
```

Comment:
Linter says it should be `const`.
Please make sure you properly configured `@web/tooling`.
You should not be able to make or update a commit when diff doesn't pass prettier and linter, except if you `--no-verify` but you should not.